js/market.js

```diff
@@ -1,59 +1,71 @@
 
 $.get('http://localhost:5000/plans')

```

Comment:
`http://localhost:5000/plans` the host needs to be a dynamic value that depends on the environment